### PR TITLE
add BytesToHumanReadable  for see_dat and see_idx

### PR DESCRIPTION
**Pull request by bingoohuang**
1. move BytesToHumanReadable to util package as public func
2. add BytesToHumanReadable of size output for  see_dat and see_idx

### CODE REVIEW

File: unmaintained/see_dat/see_dat.go [hunk 1]
```diff
@@ -31,7 +32,9 @@ func (scanner *VolumeFileScanner4SeeDat) ReadNeedleBody() bool {
 
 func (scanner *VolumeFileScanner4SeeDat) VisitNeedle(n *needle.Needle, offset int64, needleHeader, needleBody []byte) error {
 	t := time.Unix(int64(n.AppendAtNs)/int64(time.Second), int64(n.AppendAtNs)%int64(time.Second))
-	glog.V(0).Infof("%d,%s%x offset %d size %d cookie %x appendedAt %v", *volumeId, n.Id, n.Cookie, offset, n.Size, n.Cookie, t)
+	// I0529 09:56:30 12560 see_dat.go:35] 4,17f03026 offset 8 size 17214(16.81 KiB) cookie 7f03026 appendedAt 2020-05-28 10:39:56.19197 +0800 CST
```
Comment (chrislusf) [inline unmaintained/see_dat/see_dat.go]: remove this

File: unmaintained/see_idx/see_idx.go [hunk 2]
```diff
@@ -36,7 +37,8 @@ func main() {
 	defer indexFile.Close()
 
 	idx.WalkIndexFile(indexFile, func(key types.NeedleId, offset types.Offset, size uint32) error {
-		fmt.Printf("key:%v offset:%v size:%v\n", key, offset, size)
+		// key:1 offset:1 size:17214(16.81 KiB)
```
Comment (chrislusf) [inline unmaintained/see_idx/see_idx.go]: remove this

File: unmaintained/see_dat/see_dat.go (hunk 1 above)
Comment (bingoohuang) [reply unmaintained/see_dat/see_dat.go]: done

File: unmaintained/see_idx/see_idx.go (hunk 2 above)
Comment (bingoohuang) [reply unmaintained/see_idx/see_idx.go]: done

**Pull request merged**